Singer.API/Services/UserService.cs

```diff
@@ -54,11 +59,17 @@ protected UserService(ApplicationDbContext context, IMapper mapper, UserManager<
          return await base.CreateAsync(dto);
       }
 
+      private string GenerateRandomUserName(string firstName, string lastName)
+      {
+         var randomGuid = Guid.NewGuid();
+         return $"{firstName}.{lastName}.{randomGuid}@test.com";
+      }
+
       public override async Task<TUserDTO> GetOneAsync(Guid id)
       {
-         var item = await Queryable
```

Comment:
not async?

Reply:
Automapper caused an issue with IAsyncEnumerable and the SingleOrDefault.